src/app/services/cohort.service.ts

```diff
@@ -80,12 +80,6 @@ export class CohortService {
     this.updateCountsWithCurrentCohort();
   }
 
-  clearAll(): Promise<any> {
```

Comment:
So, `clearAll` in `src/app/services/mocks/cohort.service.mock.ts` can also be removed?

Reply:
it is removed from there as well

Reply:
ah, sorry, I missed that